services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/rabbitmq.py

```diff
@@ -73,10 +73,10 @@ def __init__(self, app: FastAPI, max_messages_to_send: int = 100) -> None:
         self._project_id: ProjectID = settings.DY_SIDECAR_PROJECT_ID
         self._node_id: NodeID = settings.DY_SIDECAR_NODE_ID
 
-        self._connection: aio_pika.Connection | None = None
-        self._channel: aio_pika.Channel | None = None
-        self._logs_exchange: aio_pika.Exchange | None = None
-        self._events_exchange: aio_pika.Exchange | None = None
+        self._connection: aio_pika.abc.AbstractConnection | None = None
+        self._channel: aio_pika.abc.AbstractChannel | None = None
```

Comment:
aren't you moving all these to servicelib? Or it iwll be a separate PR?

Reply:
this will be separate